webapp/src/main/resources/properties/en.properties

```diff
@@ -701,4 +701,37 @@ textUnit.gitBlameModal.close=Close
 textUnit.gitBlameModal.screenshots=Screenshots
 
 # Text for link to open screenshot modal in git-blame modal
-textUnit.gitBlameModal.screenshotModalOpen=View
\ No newline at end of file
+textUnit.gitBlameModal.screenshotModalOpen=View
+
+# Tooltip for button to open the translation history modal
+workbench.translationHistoryModal.info=Text unit history
+
+# Title of the translation history modal dialog
+workbench.translationHistoryModal.title=Text Unit Translation History
```

Comment:
Maybe just have "Translation History" here? also comment later that we could merge the title and the first line.

Reply:
Merged with title